src/OpenLoco/src/LastMapWindowAttributes.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cstdint>
+#include "Window.h"
+
+namespace OpenLoco::LastMapWindow
+{
+#pragma pack(push, 1)
+    struct LastMapWindowAttributes
+    {
+        Ui::WindowFlags lastMapWindowFlags; // 0x00526284

```

Comment:
no need to repeat lasMapWindow in this structure you would already know its the last map window by the struct.